docs_2.0/manual_EN/4.ngql-guide/12.vertex-statements/1.insert-vertex.md

```diff
@@ -0,0 +1,81 @@
+
+The `INSERT VERTEX` statement inserts a vertex or vertices into a graph space in a Nebula Graph instance.
+
+* `tag_name` denotes the `tag` (vertex type), which must be created before `INSERT VERTEX`.
+* `prop_name_list` is the property name list in the given `tag_name`.
+* `VID` is the vertex ID. In Nebula Graph 2.0.0-Alpha, only string type VID is supported. The maximum VID length is set when creating the graph space. For detail information on the maximum VID length, see [Create Space document](../9.space-statements.md/1.create-space.md).
+
+<!-- The `VID` must be unique in the graph space. The current sorting basis is "binary coding order", i.e. 0, 1, 2, ... 9223372036854775807, -9223372036854775808, -9223372036854775807, ..., -1. `VID` supports specifying ID manually, or call hash() function to generate. -->
+
+* `prop_value_list` must provide the property values according to the `prop_name_list`. If the property values do not match the data type in the tag, an error is returned. When the `NOT NULL` constraint is set for a given property, an error is returned if no property is given. When the default value for a property is `NULL`, you can omit to specify the property value. (TODO: create tag doc)
+
+`prop_value_list` must provide the values in the sequence of the properties given in `prop_name_list`. If a property value does not match the data type specified in the tag, an error is returned. When the `NOT NULL` constraint is set for a given property, an error occurs if no value is given. When the default value for a property is `NULL`, you can leave it blank and no error occurs.
+
+## Examples
+
+```ngql
+nebula> CREATE TAG t1();                   -- Create tag t1 with no property
+nebula> INSERT VERTEX t1() VALUE "10":();    -- Insert vertex "10" with no property
+```
+
+```ngql
+nebula> CREATE TAG t2 (name string, age int);                -- Create tag t2 with two properties
+nebula> INSERT VERTEX t2 (name, age) VALUES "11":("n1", 12);     -- Insert vertex "11" with two properties
+nebula> INSERT VERTEX t2 (name, age) VALUES "12":("n1", "a13");  -- Failed. "a13" is not int
+nebula> INSERT VERTEX t2 (name, age) VALUES "13":("n3", 12), "14":("n4", 8);    -- Insert two vertices
+```
+
+```ngql
+nebula> CREATE TAG t3(p1 int);
+nebula> CREATE TAG t4(p2 string);
+nebula> INSERT VERTEX  t3 (p1), t4(p2) VALUES "21": (321, "hello");   -- Insert vertex "21" with two tags.
+```
+
+A vertex can be inserted/wrote multiple times. Only the last written values can be read.
+
+```ngql
+-- insert vertex "11" with the new values.
+nebula> INSERT VERTEX t2 (name, age) VALUES "11":("n2", 13);
+nebula> INSERT VERTEX t2 (name, age) VALUES "11":("n3", 14);
+nebula> INSERT VERTEX t2 (name, age) VALUES "11":("n4", 15);
+nebula> FETCH PROP ON t2 "11";
+
++----------+---------+--------+
+| VertexID | t2.name | t2.age |
++----------+---------+--------+
+| "11"     | "n4"    | 15     |         -- Only the last version can be read
++----------+---------+--------+
```

Comment:
this is what I got: why no double quotes?

(user@nebula) [test_doc]> fetch prop on t2 "11";
+----------+---------+--------+
| VertexID | t2.name | t2.age |
+----------+---------+--------+
| 11       | n4      | 15     |
+----------+---------+--------+
Got 1 rows (time spent 4253/9470 us)

Reply:
pls update ur image